APITaxi_models/zupc.py

```diff
@@ -63,3 +67,40 @@ def top(self):
     @property
     def right(self):
         return self.bounds[2]
+
+    @staticmethod
+    def is_inactive_period():
+        inactive_filter_period = current_app.config['INACTIVE_FILTER_PERIOD']
+        hour = datetime.now().hour
+        if inactive_filter_period[0] > inactive_filter_period[1]:
+            return hour >= inactive_filter_period[0] or hour <= inactive_filter_period[1]
+        else:
+            return inactive_filter_period[0] <= hour <= inactive_filter_period[1]
+
+    @classmethod
+    def get_max_distance(cls, zupc_customer):
+        #We can deactivate the max radius for a certain zone
+        if cls.is_inactive_period:
```

Comment:
is_inactive_period is a function. This if statement always returns true. You need to call the function.

Reply:
```suggestion
        if cls.is_inactive_period():
```
nice catch, it's an habit from other languages...